pyproject.toml

```diff
@@ -55,6 +55,10 @@ addopts = [
     "--numprocesses=auto",
     "-ra",
 ]
+log_cli = true
```

Comment:
We modify the pytest configuration in order to displayed the log message. Cf pytest [log_cli](https://docs.pytest.org/en/latest/how-to/logging.html#live-logs) section.